package.json

```diff
@@ -19,32 +19,32 @@
     "postcss-value-parser": "^4.0.2"
   },
   "devDependencies": {
-    "@babel/cli": "^7.5.5",
-    "@babel/core": "^7.5.5",
-    "@babel/preset-env": "^7.5.5",
+    "@babel/cli": "^7.7.0",
+    "@babel/core": "^7.7.2",
+    "@babel/preset-env": "^7.7.1",
     "babel-plugin-istanbul": "^5.2.0",
-    "babel-preset-jest": "^24.6.0",
-    "coveralls": "^3.0.6",
-    "eslint": "^6.1.0",
+    "babel-preset-jest": "^24.9.0",
+    "coveralls": "^3.0.7",
+    "eslint": "^6.6.0",
     "eslint-config-stylelint": "^11.1.0",
     "eslint-plugin-lodash": "^6.0.0",
     "eslint-plugin-sort-requires": "^2.1.0",
     "github-contributors-list": "^1.2.3",
-    "husky": "^3.0.3",
-    "jest": "^24.8.0",
-    "jest-cli": "^24.8.0",
-    "lint-staged": "^9.2.1",
+    "husky": "^3.0.9",
+    "jest": "^24.9.0",
+    "jest-cli": "^24.9.0",
+    "lint-staged": "^9.4.3",
     "npmpub": "^5.0.0",
-    "postcss": "^7.0.17",
+    "postcss": "^7.0.21",
     "postcss-scss": "^2.0.0",
-    "prettier": "^1.18.2",
+    "prettier": "^1.19.1",
     "request": "^2.88.0",
     "rimraf": "^3.0.0",
     "semver": "^6.3.0",
-    "stylelint": "^11.0.0"
+    "stylelint": "^12.0.0"
   },
   "peerDependencies": {
-    "stylelint": "^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0"
+    "stylelint": "^10.1.0 || ^11.0.0 || ^12.0.0"
```

Comment:
This should have changed before since stylelint requires this.

Reply:
`stylelint-scss` should be compatible all the way to version 8, since I don't think that there has been any breaking changes.

Reply:
But stylelint itself sets this to `>=10.1.0` so how is this going to work? :)

Reply:
> how is this going to work?

It's quite simple actually, the peer dep there is separate from stylelint. If you use the newest version of stylelint, then the requirement might be `>=10.1.0`, but if you use an older version stylelint, then the requirement is lower, but here we can support versions 8-12.

Reply:
Your call :)

I reverted this and also updated the husky hook to get rid of a new deprecation warning.

Reply:
Maybe the support for stylelint < 10 could be dropped next time that there is a breaking change to stylelint-scss.